content/en/docs/Reference/artifacts.md

```diff
@@ -0,0 +1,151 @@
+{{% /alert %}}
+
+## Images
+
+OS images are stored in GitHub Releases, so the name of the artifact includes all the information about the image. The only exception is when the image is too big to be stored in GitHub Releases, in that case the image is stored in Quay.io.
+
+The format of the name is the following:
+
+```
+kairos-<variant>-<flavor>-<arch>-<device>-<version>.<extension>
+```
+
+Where:
+
+- `<variant>`: `core` or `standard`
+- `<flavor>`: Underlying Linux distribution e.g. `ubuntu`, `debian`, `fedora`, `alpine`, etc
+- `<arch>`: `amd64` or `arm64`
+- `<device>`: either the specific device name, e.g. `rpi64` or `generic` for generic images
+- `<version>`: the version of Kairos e.g. `v1.0.0` for Core, and can also include the K3S version e.g. `v1.0.0-k3s1` for Standard
+- `<extension>`: `iso`, `squashfs`, `ipxe`
+
+### Examples
+
+- `kairos-core-ubuntu-amd64-generic-v1.0.0.iso`
+- `kairos-standard-ubuntu-arm64-rpi64-v1.0.0-k3s1.img`
+
+## Kernel and RAM Disk Images
+
+Kernel and RAM Disk images are stored in GitHub Releases and follow a similar convention to images, but they have no extension:
+
+```
+kairos-<variant>-<flavor>-<arch>-<device>-<version>-<type>
+```
+
+Where:
+
+- `<type>`: `kernel` or `initrd`
+
+### Examples
+
+- `kairos-core-ubuntu-amd64-generic-v1.0.0-kernel`
+- `kairos-standard-ubuntu-arm64-rpi64-v1.0.0-k3s1-initrd`
+
+## iPXE Images
+
+For iPXE we deliver three types of artifacts. The first one is the iPXE script, with the ipxe extension, and the other two are iPXE bootable images, with the iso and img extensions.
+
+### Examples
+
+- `kairos-core-ubuntu-amd64-generic-v1.0.0.ipxe`
+- `kairos-core-ubuntu-amd64-generic-v1.0.0-ipxe.iso`
+- `kairos-core-ubuntu-amd64-generic-v1.0.0-ipxe-usb.img`
+
+## Reports
+
+Reports are also stored in GitHub Releases and follow a similar convention to images, but they include the name of the report:
+
+```
+kairos-<variant>-<flavor>-<arch>-<device>-<version>-<report>.<extension>
+```
+
+Where:
+
+- `<report>`: `trivy`, `sbom`, `grype`, etc
+
+### Examples
+
+- `kairos-core-alpine-arm64-generic-v1.0.0-trivy.sarif`
+- `kairos-core-alpine-arm64-generic-v1.0.0-grype.json`
+
+## Container Images
+
+{{% alert title="Warning" %}} 
+The repositories are not yet available, they will be published soon.
+{{% /alert %}}
+
+Container images are stored in Quay.io and follow the following convention:
+
+```
+domain/kairos/<variant>-<flavor>-<device>:<version>
+```
+
+This nomenclature for container images lacks some information for the following reasons:
+
+1. Docker repositories can be multi-arch, so the same image can be used in different architectures and therefore the name does not include the architecture.
+2. The version is not included in the image name, but in the tag.
+3. The name Kairos is already part of the repository name, so it is not included in the image name.
+
+### Examples
+
+- quay.io/kairos/core-ubuntu-generic:v1.0.0
```

Comment:
@jimmykarily according to the discussion today we will use tags for the support of the image. Do you know how that looks in the end? Maybe something like:

`quay.io/kairos/core-ubuntu-generic:community-v1.0.0`

and if that is the case, should we also move the model into the tag?

`quay.io/kairos/core-ubuntu-generic:community-generic-v1.0.0`

Reply:
Not "tags", "labels". Like these: https://docs.docker.com/engine/reference/builder/#label